api/src/main/java/org/openmrs/Diagnosis.java

```diff
@@ -0,0 +1,159 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+/**
+ * Diagnosis class defines the identification of the nature of an illness or other problem 
+ * by examination of the symptoms during an encounter (visit or interaction of a patient with a healthcare worker).
+ */
+
+public class Diagnosis extends BaseChangeableOpenmrsData{
```

Comment:
It would also be great to format this by putting some space after the base class but before {